src/main/java/com/wynntils/modules/core/instances/MainMenuButtons.java

```diff
@@ -146,4 +146,4 @@ private static void doAction() {
         }
     }
 
-}
+}
```

Comment:
Unintentional newline removal?
```suggestion
}

```